service-commands/scripts/A

```diff
@@ -1,3 +1,297 @@
-#!/bin/bash
-cd $PROTOCOL_DIR/service-commands
-node scripts/setup.js $@
\ No newline at end of file
+#!/usr/bin/env python3
```

Comment:
is there a reason why this script is called `A`? doesn't seem very descriptive

Reply:
@dmanjunath @raymondjacobson thoughts on calling this `audius-cli`?

Reply:
@vicky-g 
![image](https://user-images.githubusercontent.com/295938/129251702-7883d8aa-5517-4bd9-b612-9f30d1e4b3fd.png)